scripts/verify-pkg-main.js

```diff
@@ -75,14 +40,14 @@ function verifyPath(packageJson, jsonPath, packagePropertyKey) {
   let isInvalid = false;
   if (!isAtRoot && packageJsonPropPath.indexOf('dist') === -1) {
     isInvalid = true;
-    console.error(`${jsonPath} ${packagePropertyKey} property does not reference a file under dist`);
+    logError(`${jsonPath} ${packagePropertyKey} property does not reference a file under dist`);
```

Comment:
This is now going to exit as soon as a single invalid path is found, which is less helpful than printing all issues out at once so that someone can fix them all at once before running it again.

Reply:
Nevermind, I just now realized the change made to `logError`, which is ridiculously subtle...